IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -235,129 +216,148 @@ - (void)executeRequestImpl:(MSIDRequestCompletionBlock)completionBlock
             }
         }
     }
+    
+    [self fetchCachedRefreshTokenWithCompletionHandler:^(MSIDBaseToken<MSIDRefreshableToken> *refreshToken, MSIDRefreshTokenTypes tokenType, NSError *error) {
+        if (error)
```

Comment:
I think we should check for refresh token and not error here, since when there's no refresh token in cache, there'll be no error, but we shouldn't proceed.

Reply:
fixed